doc/man/task-sync.5.in

```diff
@@ -5,21 +5,97 @@ task-sync \- A discussion and tutorial for the various task(1) data
 synchronization capabilities.
 
 .SH INTRODUCTION
-Taskwarrior has several sync options, both external and built in.  If you wish
-to sync your data, choose one method only; mixing methods is going to lead to
-problems.  Each of the methods discussed have their own strengths.
+
+Taskwarrior can "synchronize" your tasks to a server. This has a few benefits:
+.br
+  - Makes your tasks accessible from multiple systems.
+.br
+  - Provides a backup of your tasks.
+.br
+  - Allows internal storage to be reused.
+
+.SH LOCAL SYNCHRONIZATION
+Taskwarrior can sync your tasks locally. While this setup does not make your
+tasks accessible from multiple systems, it does provide a convenient backup
+and allow internal storage to be reused. If you later decide to sync to a
+server, the transition is easy.
+
+Set 'sync.local.server_dir' to the directory where the synchronized data should
+be stored:
+
+    $ task config sync.local.server_dir /path/to/sync
+
+Then run 'task sync' to synchronize to this local "server".
+
+.SH SERVER SYNCHRONIZATION
+Configure each instance of Taskwarrior which you would like to keep in sync
+with the same settings.  You will need the following information from the
+adminisrator of the server:
+.br
+  - The server's URL ("origin", such as "https://tw.example.com")
+.br
+  - A client key ("client_key") identifying your tasks
+.br
+  - An encryption secret ("encryption_secret") used to encrypt and decrypt your
+    tasks.
+
+Configure Taskwarrior with these details:
+
+    $ task config sync.server.origin            <origin>
+    $ task config sync.server.client_key        <client_key>
+    $ task config sync.server.encryption_secret <encryption_secret>
+
+Then run 'task sync' to synchronize the tasks to the server.
+
+To add a new, empty Taskwarrior installation to the set of replicas, such as
+when setting up a new system, run 'task sync' in that installation before
+adding any tasks.
+
+It is easy to transition from local to server synchronization: simply change
+all replicas to use the same "sync.server.*" configuration.
```

Comment:
I don't necessarily think this is obvious -- I'd like users to know that they don't need to find a server _today_, but can start tracking tasks and set up sync later.

Where would be a better place for it?

Reply:
I think you could elaborate on it more in the section on "Local Synchronization" rather than foreshadowing:

> If you later decide to sync to a server, the transition is easy.

Maybe something like:

> If you later decide to sync to a server, the transition is as simple as changing all replicas to use the server synchronization settings explained in the next section.

Reply:
So can this sentence be dropped now?